global-cli/index.js

```diff
@@ -166,3 +169,13 @@ function checkNodeVersion() {
     process.exit(1);
   }
 }
+
+function isGitHubBoilerplate(root) {
+  var validFiles = [
+    '.DS_Store', 'Thumbs.db', '.git', '.gitignore', 'README.md', 'LICENSE'
```

Comment:
this essentially makes it impossible to use another collaboration/versioning product eg subversion, or some other IDE such as eclipse to register the project. I am trying to co-locate my project in eclipse and use subversion for collaboration. Eclipse creates a ".project" file, and subversion creates ".svn" directories.